lib/glueby/contract/active_record.rb

```diff
@@ -3,6 +3,11 @@
 module Glueby
   module Contract
     module AR
+      # Add prefix to activerecord table names
+      def self.table_name_prefix
+        'glueby_contract_'
```

Comment:
It is better to unify the prefix only with `glueby_`.

Reply:
I see. I will modify it.